tensorflow_io/image/python/ops/image_dataset_ops.py

```diff
@@ -21,39 +21,9 @@
 from tensorflow import dtypes
 from tensorflow.compat.v1 import data
 from tensorflow_io import _load_library
+from tensorflow_io.core.python.ops import data_ops as data_ops
```

Comment:
nit: could we just use `from tensorflow_io.core.python.ops import data_ops` without `as data_ops`?